force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,13 +66,25 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
-        Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        // Check if either num1 or num2 is null
+        if (num1 == null || num2 == null) {
+            return null;
+        }
+        
+        // Compare num1 and num2, and return the larger one
+        if(num1 > num2) {
+            return num1;
+        }else if (num1 < num2){
+            return num2;
+        }else if (num1 == num2){
+            return num1;
+        }else{
+            return null; // Return null if both are equal
+        }
```

Comment:
Try to use the Math.max() method to do this one for more practice
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max